src/main/java/uk/gov/pay/adminusers/client/ledger/model/LedgerTransaction.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.pay.adminusers.client.ledger.model;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
```

Comment:
Don't think the @JsonInclude annotation is needed - this is just needed if we're serialising this class to a response, which we're not doing.

Reply:
We shall remove it in the next iteration.